app/controllers/transition_landing_page_controller.rb

```diff
@@ -41,7 +41,7 @@ def show_comms?
   end
 
   def show_countdown?
-    false
+    show_variant? || false
```

Comment:
Do we need the `|| false`? It confused me a little

Reply:
I put it in as I wasn't sure sure where [this error](https://github.com/alphagov/govuk_ab_testing/blob/75d837afa3fc2bde2d06b6a93f5a6db29b42b013/lib/govuk_ab_testing/requested_variant.rb#L47) would bubble up to if the variant in the header isn't A B or Z, in this method:

```
def show_variant?
   page_under_test? && && ab_test_variant.variant?("B")
end
```
But it actually doesn't matter because even if `ab_test_variant.variant?("B")` returns `nil`, `true && nil` is `nil`, so we won't render the component.

So the TLDR is.... we don't need the `|| false` . I shall delete it.